DemoApp/StreamChat/Components/DemoChatChannelListRouter.swift

```diff
@@ -298,6 +298,19 @@ final class DemoChatChannelListRouter: ChatChannelListRouter {
                     }
                 }
             }),
+            .init(title: "Mute channel with expiration", isEnabled: canMuteChannel, handler: { [unowned self] _ in
+                self.rootViewController.presentAlert(title: "Enter expiration", textFieldPlaceholder: "Milliseconds") { expiration in
```

Comment:
nit: It may be better to use seconds as they are more human readable format for time

Reply:
Makes sense, updated 👍